src/style.css

```
@@ -0,0 +1,88 @@
+body {
+  margin: 0;
+}
+
+/* .container {
+  margin: 32px 22px;
+  width: 80px;
+  height: 78px;
+} */
+
+.stars {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  width: 80px;
+  height: 12px;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  width: 16px;
+  height: 12px;
+  list-style: none;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 li:nth-child(1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 li:nth-child(1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 li:nth-child(2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 li:nth-child(1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 li:nth-child(2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 li:nth-child(3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 li:nth-child(1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 li:nth-child(2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 li:nth-child(3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 li:nth-child(4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 li:nth-child(1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 li:nth-child(2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 li:nth-child(3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 li:nth-child(4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 li:nth-child(5) {
+  background-image: url(images/star-active.svg);
+}
```

do not hardcode each active star.
it can be more flexible solution. 
also use classes instead tags for styling. and if  several selectors should have same styles - they are can be styling throw comma.
```suggestion
.stars--1 .stars__star:nth-child(-n+1),
.stars--2 .stars__star:nth-child(-n+2),
.stars--3 .stars__star:nth-child(-n+3),
.stars--4 .stars__star:nth-child(-n+4),
.stars--5 .stars__star:nth-child(-n+5) {
  background-image: url("./images/star-active.svg");

```